test/integration/associations/scope.test.js

```diff
@@ -357,19 +357,25 @@ describe(Support.getTestDialectTeaser('associations'), () => {
               this.postC.getTags()
             ]);
 
-            expect(postACategories.length).to.equal(1);
-            expect(postATags.length).to.equal(1);
-            expect(postBCategories.length).to.equal(1);
-            expect(postBTags.length).to.equal(1);
-            expect(postCCategories.length).to.equal(1);
-            expect(postCTags.length).to.equal(1);
-
-            expect(postACategories[0].get('type')).to.equal('category');
-            expect(postATags[0].get('type')).to.equal('tag');
-            expect(postBCategories[0].get('type')).to.equal('category');
-            expect(postBTags[0].get('type')).to.equal('tag');
-            expect(postCCategories[0].get('type')).to.equal('category');
-            expect(postCTags[0].get('type')).to.equal('tag');
+            // Flaky test: randomly one of the value on B will be 0 sometimes, for
+            // now no solution. Not reproducible at local or cloud with logging enabled
+            expect([
+              postACategories.length,
+              postATags.length,
+              postBCategories.length,
+              postBTags.length,
+              postCCategories.length,
+              postCTags.length
+            ]).to.eql([1, 1, 1, 1, 1, 1]);
```

Comment:
Shouldn't this need a deep equality comparison?

Reply:
`eql` is same as deep equal in mocha...
https://stackoverflow.com/questions/36798993/what-is-the-difference-between-equal-and-eql-in-chai-library

Yes I also feel use `.deep.equal` is better for next step ( switch to jest), let me fix it very quick.

Reply:
Forget about that `is better` above - since anyway we need to change the `.to.` later. This should be good to go.

Reply:
Interesting. I didn't know that and always used `to.deep.equal`:)